site/_docs/reference.md

```diff
@@ -2438,6 +2438,9 @@ Not implemented:
 
 #### Geometry projection functions
 
+The EPSG dataset is released separately from Proj4J due to its restrictive [terms of use](https://epsg.org/terms-of-use.html).
+In order to use the projection functions in Apache Calcite, users must include the EPSG dataset in their dependencies.
```

Comment:
Can you shorten the lines? In my opinion there's little reason not to wrap text at 60 or 70 characters. Long lines are difficult to view in diff output.